contribute/ISSUE_TRIAGE.md

```diff
@@ -103,80 +104,80 @@ Should explain what happened, what was expected and how to reproduce it together
+When submitting an enhancement request we ask that users focus on the problem they'd like to solve and why it’s a problem rather than focusing on the solution itself. To facilitate these objectives, the feature requests template includes the following:
 
 - What would you like to be added?:
 - Why is this needed (describe your use case and goals)\*\*?:
 
-#### Accessibility issues
+##### Accessibility issues
 
-This is a mix between a bug report and enhancement request but focused on accessibility issues to help make Grafana improve keyboard navigation, screen-reader support and being accessible to everyone. The report should include relevant [WCAG criteria](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0), if applicable.
+This is a mix between a bug report and enhancement request but focused on accessibility issues to help make Grafana improve keyboard navigation, screen-reader support, and general accessibility. The report should include relevant [WCAG criteria](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0), if applicable.
 
 Grafana Labs is dedicated to improving our graphical user interfaces and overall experience so that our product becomes usable and accessible for people with disabilities as well as anyone else. Learn more about Grafana's commitment to [A11y](https://grafana.com/accessibility/) (accessibility).
 
-#### Support requests
+##### Support requests
 
 In general, if the issue description and title is perceived as a question no more information is needed. See how to categorize these requests [here](#support-requests-1).
 
-### Good practices
+#### Good practices
 
-To make it easier for everyone to understand and find issues they're searching for it's suggested as a general rule of thumbs to:
+To make it easier for everyone to understand and find issues a good rule of thumbs is to:
 
-- Make sure that issue titles are named to explain the subject of the issue, has a correct spelling and doesn't include irrelevant information and/or sensitive information.
-- Make sure that issue descriptions doesn't include irrelevant information, information from template that haven't been filled out and/or sensitive information.
-- Do your best effort to change title and description or request suggested changes by adding a comment.
+- Make sure that issue titles are named to explain the subject of the issue, are spelled correctly, and don't include irrelevant or sensitive information.
+- Make sure that issue descriptions don't include irrelevant information, information from a template that hasn't been filled out, or sensitive information.
+- Do your best effort to change the title and description or request suggested changes by adding a comment.
 
-> **Note:** Above rules is applicable to both new and existing issues of the Grafana project.
+> **Note:** The preceding rules apply to both new and existing issues of the Grafana project.
```

Comment:
I think this should be deleted, but if you want to keep it, please use the admonition shortcode.